Makefile

```diff
@@ -188,3 +191,11 @@ index-push:
 
 .PHONY: deploy-olm
 deploy-olm: bundle-build bundle-push index-build index-push
+
+.PHONY: deploy-olm-testing
+deploy-olm-testing: 
+	sed -i 's/keda/keda-test/' bundle/metadata/annotations.yaml
+	sed -i 's/keda.v2.0.0/keda-test.v2.0.0/' bundle/manifests/keda.clusterserviceversion.yaml
```

Comment:
The version shouldn't be hardcoded, you should use the variable defined at the top